tethys_gizmos/gizmo_options/map_view.py

```diff
@@ -318,8 +318,8 @@ class MapView(TethysGizmoOptions):
     local_url = 'tethys_gizmos/vendor/openlayers/{version}/ol.{ext}'
 
     def __init__(self, height='100%', width='100%', basemap=None, view={'center': [-100, 40], 'zoom': 2},
-                 controls=[], layers=[], draw=None, legend=False, attributes={}, classes='', disable_basemap=False,
-                 feature_selection=None):
+                 controls=[], layers=[], models=[], primitives=[], draw=None, legend=False, attributes={}, classes='',
+                 disable_basemap=False, feature_selection=None):
```

Comment:
Why are models and primitives being added to the map_view? This view is OpenLayers powered and doesn't support models or primitives.

Reply:
Remove `models` and `primitives` from the `MapView` gizmo.

Reply:
Changes to map_view.py still need to be reverted.